scripts/migrate.ts

```diff
@@ -0,0 +1,61 @@
+import mysql, { Connection } from 'mysql2/promise'
+import path from 'node:path'
+import fs from 'node:fs'
+import Postgrator from 'postgrator'
+
+interface PostgratorResult {
+  rows: any;
+  fields: any;
+}
```

Comment:
```suggestion
interface PostgratorResult {
  rows: RowDataPacket[];
  fields: FieldPacket[];
}
```

Reply:
Unfortunately this is enforced by the library I use...
```ts
  export interface BaseOptions {
    schemaTable?: string;
    validateChecksums?: boolean;
    migrationPattern?: string;
    newline?: string;
    execQuery?: (query: string) => Promise<{ rows: any[] }>;
    execSqlScript?: (sqlScript: string) => Promise<void>;
  }
```

Do you know how to extends that? Anyway, this not important for the scripts but if you know how to not use any tell me.